packages/flutter/lib/src/services/flutter_version.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// [FlutterVersion] contains various meta information about whith which Flutter build this app was compiled.
+/// This contains the information that is output by executing `flutter --version`.
+/// 
+/// See also:
+///  - [Platform.version](https://api.dart.dev/stable/dart-io/Platform/version.html)
```

Comment:
This is a markdown link so that it doesn't include `dart:io`. Is there a way to use a dartdoc reference without importing `dart:io`?